DSP Accel Library/Testing and Images/TestingLibraryScripts.m

```diff
@@ -0,0 +1,129 @@
+%% TESTING ONLY, USER DEFINED VARS
+clear;
```

Comment:
Also add "close all"

Reply:
This file (TestingLibraryScripts.m) was the init script of the mask I created for the PLUT, I'm not sure how close all would mess with the Simulink model. I also set up any figures to a preset number so running the code multiple times wouldn't flood your screen with charts.
I can make the change if you still think it's good

Reply:
The command should just close all plots, so I wouldn't think it will mess with the Simulink model.  Since you number your plots, it will just create new figures if the corresponding figure number doesn't exist when the script reaches the plotting commands.  If it does mess with the model, I'd say make a note at the beginning of the script about what scripts need to be run before this one and list other requirements.